etc/scripts/get-stack.sh

```diff
@@ -609,7 +609,7 @@ try_install_pkgs() {
 
 # Install packages using apt-get
 apt_get_install_pkgs() {
-  if ! dpkg-query -W "$@"|grep -v '^\S\+\s\+.\+$' > /dev/null; then
+  if dpkg-query -W "$@" > /dev/null; then
```

Comment:
Any particular reason you changed this?  I believe the reason for the `grep` was to make it detect if only _some_ of the required packages are missing.  That said, it could be more intelligent by only trying to install the actually missing dependencies (rather than all of them, even if only some are missing).

Reply:
See my comment above. This is somehow broken in the version of `/bin/sh` that MX uses (which is based on Debian stable). I can try to figure out a better way to handle it.

Reply:
Ok, sorry I didn't read the description well enough.  I'm OK with the revert, but how about adding a comment with what you reverted and why (and a link to this discussion) so that if we change it again we won't accidentally break MX.

Reply:
I added a comment and verified this in both Debian and MX.